CHANGELOG.md

```diff
@@ -1,5 +1,17 @@
 # Changelog
 
+## [12.1.0](https://github.com/theforeman/puppet-candlepin/tree/12.1.0) (2021-10-29)
+
+[Full Changelog](https://github.com/theforeman/puppet-candlepin/compare/12.0.1...12.1.0)
+
+**Breaking changes:**
```

Comment:
Good catch, I just looked at the PR wording not the section it landed in. So I think my initial reaction was https://github.com/theforeman/puppet-candlepin/pull/207 is more of an enhancement since its related to newer Candlepin's even though it changes the interface. But I'm happy to go either way.

Reply:
Very strictly speaking it's an API change, but I don't think we rely on it in our modules so I'd consider a non-breaking change this time.